### PR TITLE
fix #11 - updating dependencies for jdepend to a fork supporting java 8

### LINKED ISSUE #11: Unknown constant: 18
Hi,
I'm getting the output `Unknown constant: 18` multiple times when checking some of my Java 8 projects.

Example output:
```
...
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-no-package-cycles) @ xxx ---
[INFO] Adding directory /home/daniel/xxx/target/classes for package cycles search.
[INFO] Adding directory /home/daniel/xxx/target/test-classes for package cycles search.

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18

Unknown constant: 18
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
...
```

### PR DESCRIPTION
**Pull request by galan**
Updated jdepend to `guru.nidi:jdepend:2.9.5`, also updated other dependencies.

### CODE REVIEW

File: src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java [hunk 1]
```diff
@@ -24,38 +24,45 @@
...
 		helper.setClassesDir(new File(targetFolder.toURI()));
 		try {
 			rule.execute(helper);
 			fail("expected EnforcerRuleException");
-		} catch (EnforcerRuleException e) {
+		}
+		catch (EnforcerRuleException e) {
 			// using assertEquals to get a nice comparison editor in eclipse
-			assertEquals(getExpectedOutput(expectedOutput), e.getMessage());
+			assertEquals(getExpectedOutput(expectedOutput), e.getMessage() + "\n");
```
Comment (marcphilipp) [inline src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java]: Why did you change this file?

File: src/test/resources/junit-expected-output.txt [hunk 2]
```diff
@@ -16,7 +16,7 @@ Package-cycle found involving junit.extensions, junit.framework, junit.runner, o
         org.junit.runners (FixMethodOrder)
     org.junit.experimental.theories depends on:
         org.junit (Theories$TheoryAnchor, Theories$TheoryAnchor$1, Theories$TheoryAnchor$2)
-        org.junit.experimental.theories.internal (Theories$TheoryAnchor, Theories$TheoryAnchor$1, Theories$TheoryAnchor$1$1, Theories$TheoryAnchor$2)
+        org.junit.experimental.theories.internal (FromDataPoints, Theories$TheoryAnchor, Theories$TheoryAnchor$1, Theories$TheoryAnchor$1$1, Theories$TheoryAnchor$2)
```
Comment (marcphilipp) [inline src/test/resources/junit-expected-output.txt]: Why is this change necessary?

File: src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java (hunk 1 above)
Comment (galan) [inline src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java]: Tests will fail in IntelliJ if not. Will work via mvn build with or without linebreak.

File: src/test/resources/junit-expected-output.txt (hunk 2 above)
Comment (galan) [inline src/test/resources/junit-expected-output.txt]: Because the output changed with the new version.

File: src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java (hunk 1 above)
Comment (davidburkhart) [inline src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java]: @galan Are you working on Windows? If you take a close look at the method getExpectedOutput you will see that \r is beeing replaced with nothing. In my opinion \n should be replaced there as well to be consistent with the existing code.

Comment (galan) [inline src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java]: I'm working on Linux

Comment (galan) [inline src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java]: I revert my previous statement, the build fails on the command line via `mvn` as well. I also tested eclipse, with the same result. Checked the pom.xml as well. Turns out that the "intelligent" IDE IntelliJ has modified the encoding of the files. One reason I usually stick to eclipse. Anyways I reverted the related files, see new commit.

**Comment by marcphilipp** [comment]
@davidburkhart Can you do a release?

**Comment by davidburkhart** [comment]
Trying ;)
